src/trackers/TOS.py

```
@@ -0,0 +1,301 @@
+
+        specified_dir_path = os.path.join(search_dir, '*.nfo')
+        nfo_files = glob.glob(specified_dir_path)
+
+        if nfo_files:
+            async with aiofiles.open(nfo_files[0], 'rb') as f:
+                nfo_bytes = await f.read()
+            files['nfo'] = ("nfo_file.nfo", nfo_bytes, "text/plain")
+
+        return files
+
+    async def upload(self, meta, disctype):
```

Ruff (0.14.10)

Unused method argument: `disctype`

(ARG002)